.github/workflows/codeql.yml

```diff
@@ -21,20 +21,29 @@ jobs:
 
     steps:
       - name: Check out repository
-        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 #v3.5.3
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 #v4.1.1
         with:
           fetch-depth: 2
 
+      - name: Download benchmark bundle
+        env:
+           GH_TOKEN: ${{ github.token }}
+        run: |
+           gh release download -R octo-org/codeql-bundle --pattern 'codeql-bundle.tar.gz'
```

Comment:
Hmm. This line seems to be erring out.

I think `octo-org` might just be an example organization, or something. I think this was an example of how to add your own custom CodeQL rules to the CodeQL Standard Library. I'm still trying to figure out where one gets the latter. Is it this? https://github.com/github/codeql-coding-standards

Reply:
Oh, OK. Looks like it's this: https://github.com/github/codeql-action/releases

Reply:
I was copying their example....guess its more example than working functionality...

Reply:
okay - updated to use the right things for those downloads.